am-ui/webcomponents/create-ad-webcomponent/src/components/PreviewAd.vue

```diff
@@ -1,126 +1,128 @@
+      break;
+    }
+    case 'priceunit': {
+      return t(`categories.accomodation.priceUnit.options.${field.value}`);
+      break;
+    }
+    case 'availability': {
+      return t(`categories.accomodation.availability.options.${field.value}`);
+      break;
+    }
+    case 'date': {
+      const splittedDate = field.value.split('-');
+      return `${splittedDate[2]}.${splittedDate[1]}.${splittedDate[0]}`;
+      break;
     }
-    /**
-     * Retrieves the Title property from the Field.
-     * @param {[{}]} field - contains all the field properties
-     */
-     function getFieldTitle(field, category) {
-       switch(field.key.toLowerCase()) {
-           case "priceunit":
-              return t('categories.accomodation.priceUnit.title');
-              break;
-           case "availability":
-              return t('categories.accomodation.availability.title');
-              break;
-           default:
-              return t(`categories.${category}.${field.key}`);
-              break;
-       }
-     }
+    case 'furnished': {
+      return field.value ? t('affirmative') : t('negative');
+      break;
+    }
+    case 'temporary': {
+      return field.value ? t('affirmative') : t('negative');
+      break;
+    }
+    default: {
+      return field.value;
+      break;
+    }
+  }
+}
+/**
+ * Retrieves the Title property from the Field.
+ * @param {[{}]} field - contains all the field properties
+ * @param {[{}]} category - the category of the field
+ * @return {FieldTitle} Tranlated field title
+ */
+function getFieldTitle(field, category) {
+  switch (field.key.toLowerCase()) {
+    case 'priceunit':
+      return t('categories.accomodation.priceUnit.title');
+      break;
+    case 'availability':
+      return t('categories.accomodation.availability.title');
+      break;
+    default:
+      return t(`categories.${category}.${field.key}`);
+      break;
+  }
+}
 
-     function submit(){
-       const fields = [];
-       props.fieldsToPreview.forEach((field) => {
-            fields.push({
-              field_type_id: field.field_id,
-              value: field.value
-            });
-        });
-       emit('submit', fields, props.images, props.thumbnail);
-     }
+/**
+ * Goes back from preview to create view with the form data
+ */
+function submit() {
+  const fields = [];
+  props.fieldsToPreview.forEach((field) => {
+    fields.push({
+      field_type_id: field.field_id,
+      value: field.value
+    });
+  });
+  emit('submit', fields, props.images, props.thumbnail);
+}
 
-     function back(){
-       const fields = [];
-        props.fieldsToPreview.forEach((field) => {
-            fields.push({
-              field_id: field.field_id,
-              field_type_definition_id: field.field_type_definition_id,
-              key: field.key,
-              value: field.value,
-              field_options: field.field_options
-            });
-        });
-        emit('back', fields);
-     }
+/**
```

Comment:
Am I understanding wrong or is this description wrong? Nothing is published on back(). Oh now that I read it maybe it's just mixed up with the submit method

Reply:
yeap! They were mixed up with eachother, merci. Fixed!